tests/baselines/reference/user/content-disposition.log

```diff
@@ -0,0 +1,30 @@
+Exit Code: 1
+Standard output:
+node_modules/content-disposition/index.js(11,5): error TS1212: Identifier expected. 'public' is a reserved word in strict mode.
+node_modules/content-disposition/index.js(19,5): error TS1212: Identifier expected. 'private' is a reserved word in strict mode.
+node_modules/content-disposition/index.js(27,5): error TS1212: Identifier expected. 'private' is a reserved word in strict mode.
+node_modules/content-disposition/index.js(34,5): error TS1212: Identifier expected. 'private' is a reserved word in strict mode.
+node_modules/content-disposition/index.js(42,5): error TS1212: Identifier expected. 'private' is a reserved word in strict mode.
+node_modules/content-disposition/index.js(50,19): error TS1127: Invalid character.
```

Comment:
need to look at this too

Reply:
ha ha oh no it's the scanner. @weswigham we are now giving two errors:

```js
/**
 * RegExp to match quoted-pair in RFC 2616
 *
 * quoted-pair = "\" CHAR
 * CHAR        = <any US-ASCII character (octets 0 - 127)>
 * @private
 */
```

1. On `"\"`
2. `@private`

Broken by #32716

Reply:
Filed #32756 to track this